src/index.html

```diff
@@ -6,7 +6,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+
+    <div class="calendar page__calendar calendar--start-day--wed calendar--month-length--30">
+
+      <div class="day calendar__day-1"></div>
+      <div class="day calendar__day-2"></div>
```

Comment:
no need make a day as a separate BEM block, it's almost impossible that you will use it separately from the calendar